App.tsx

```diff
@@ -64,6 +71,29 @@ export default function App() {
     void prepare()
   }, [])
 
+  const handleShare = useCallback(async (item?: SharedItem) => {
```

Comment:
this should not be in the app.tsx. I would expect this logic to be in the SharedDataProvider class

Reply:
also see other comment

Reply:
attempted a fix.

I see following error.
C:\dev\workspaces\ws1\ssi-mobile-wallet\node_modules\react-native\Libraries\Core\ExceptionsManager.js:149 The action 'NAVIGATE' with payload {"name":"VerificationCode","params":{"credentialName":"OpenBadgeCredential"}} was not handled by any navigator.

Do you have a screen named 'VerificationCode'?

Reply:
Fixed the navigation issue.